src/binaryen-c.h

```diff
@@ -356,6 +390,12 @@ void BinaryenRemoveExport(BinaryenModuleRef module, const char* externalName);
 // Globals
 
 BinaryenImportRef BinaryenAddGlobal(BinaryenModuleRef module, const char* name, BinaryenType type, int8_t mutable_, BinaryenExpressionRef init);
```

Comment:
hmm, this was wrong before, `AddGlobal` should return a global, not an import. Let's fix that here and in the new places it is used in this PR.